source/css/sass/patterns/templates/content-grid.scss

```diff
@@ -0,0 +1,91 @@
+@import "../../functions/grid";
+@import "../../mixins/grid";
+@import "../../mixins/media-query";
+@import "../../variables/grid";
+
+.content-grid {
+  --primary-column-width: #{column-width($grid-main_column_count)};
+
+  @include base-grid($grid-max_width);
+  grid-template-areas:
+    ". tertiary ."
+    ". primary  .";
+  grid-template-columns: [full-start] 1fr [main-start] var(--primary-column-width) [main-end] 1fr [full-end];
+  grid-column-gap: var(--column-gap);
+
+  @include mq($from: wide) {
+    $primary-columns: $grid-main_column_count - 2;
+    $secondary-columns: floor(($grid-main_column_count - $primary-columns) / 2);
+    $tertiary-columns: $grid-main_column_count - $primary-columns - $secondary-columns;
+    --primary-column-width: #{column-width($primary-columns)};
+    --secondary-column-width: #{column-width($secondary-columns)};
+    --tertiary-column-width: #{column-width($tertiary-columns)};
+    grid-template-areas:
+      ". . tertiary . ."
+      ". . primary  . .";
+    grid-template-columns: [full-start] 1fr [main-start] var(--tertiary-column-width) var(--primary-column-width) var(--secondary-column-width) [main-end] 1fr [full-end];
```

Comment:
There's a bit of confusion with the `secondary/tertiary` areas and the columns here. Wondering if the `column-width` variables should just be renamed `one`, `two` and `three`?